package.json

```diff
@@ -100,6 +100,7 @@
       "toggle-off",
       "traffic-cone",
       "trash-can",
+      "up-right-and-down-left-from-center",
```

Comment:
Such a weird icon name.

Reply:
The alias for it, `expand-alt`, makes so much more sense.

Reply:
font awesome was naming icons based on an assumed function like expand/contract.. then they had a shift in thinking and now are just describing the icon absent function.